capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="event.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="hidden" id="organizationID" name="organizationID"> 
+        <br><br>
+
+        <label for="organizationName"> Organization Name: </label>
+        <input type="text" id="organizationName" name="organizationName">
+        <br><br>
+
+        <label for="university"> University: </label>
```

Comment:
Optional, for future reference: grouping a set of fields under `<div class='field-group-or-something'> ... </div>` gives you more styling flexibility than hardcoded line breaks.

Reply:
I will keep this in mind!